lib/ravenex/notifier.ex

```
@@ -12,11 +12,13 @@ defmodule Ravenex.Notifier do
   }
 
   def notify(error, options \\ []) do
-    case get_dsn do
-      dsn when is_bitstring(dsn) ->
-        build_notification(error, options)
-        |> send_notification(dsn |> parse_dsn)
-      :error -> :error
+    if proceed?(Application.get_env(:ravenex, :ignore), error) do
+      case get_dsn do
+        dsn when is_bitstring(dsn) ->
+          build_notification(error, options)
+          |> send_notification(dsn |> parse_dsn)
+        :error -> :error
+      end
```

Maybe we can add an else clause with a `Logger.info` that we're ignoring the error? I think it would be nice to have that in our logs if we're not going to log it to Sentry.

---

Sounds good, changing that